src/main/java/org/synyx/urlaubsverwaltung/person/Person.java

```diff
@@ -195,17 +195,4 @@ public void setId(Integer id) { // NOSONAR - needed for setting ID in form
 
         super.setId(id);
     }
-
-
-    @Override
-    public String toString() {
-
-        return new ToStringBuilder(this, ToStringStyle.SHORT_PREFIX_STYLE).append("id", getId())
```

Comment:
Hier vllt nur eine toString mit der id?

Reply:
Habe es gerade nochmal genauer nachgeschaut. `AbstractPersistable` hat natürlich seine eigene toString(), welche nur die id rausgibt. Hier würde ich aber in der `Person` explizit sein + Test, sodass wir immer wissen was passiert. Falls jemand die Klasse nicht mehr von  `AbstractPersistable` erben lässt/ändert und mit @Id arbeitet haben wir dann evtl ein Problem. Mit der expliziten toString können wir dies umgehen.